perfplot/main.py

```diff
@@ -49,16 +66,23 @@ def __init__(
             self.timings = self.timings[order]
             self.labels = [self.labels[i] for i in order]
             self.colors = [self.colors[i] for i in order]
+
+        if automatic_scale:
+            # Makes timings more readable on plot
+            self._scale_timings()
+        else:
+            self.timings *= 1e-9  # Converting from ns to s
+            self.timings_unit = "s"
```

Comment:
That's a bit confusing. You specify a `timings_unit`, but it `automatic_scale` is `False`, it's reset. It's better to make `automatic_scale` and `timings_unit` _one_ variable, e.g., `time_unit`. If `None`, make it automatic. The conditional could look like
```
self.time_unit = self._automatic_time_unit() if time_unit is None else time_unit
```

Reply:
Hey, thanks for the feedback, indeed it is confusing. I shouldn't be coding so late at night. With your suggestion I was able to come up with a new implementation: 21588a07bcaac2828a40a5faacb47f450f063a5a